urth_dash_js/notebook/dashboard-common/error-log.js

```diff
@@ -1,43 +0,0 @@
-/**
- * Copyright (c) Jupyter Development Team.
- * Distributed under the terms of the Modified BSD License.
- */
-
-/**
- * Registers an error handler which prints errors to the JS console.
- */
-define([
-    'require',
-    'jquery'
-], function(
-    require,
-    $
-) {
-    'use strict';
-    var enabled = false;
-    return {
-        /**
-         * Show IOPub errors in the JS console.
-         * @param  {object} IPython    "global" IPython singleton
-         */
-        enable: function(IPython) {
```

Comment:
`jupyter-js-output-area` currently handles error by adding it to the page. We figured that was good enough for now. We can't re-use error-log.js because the APIs changed significantly and there's not central error handler we can key off of. In the future, we may revisit this to see if we can simply post errors in the console -- but they are not hidden in any way with this patch.